website/content/docs/agent/caching/index.mdx

```diff
@@ -57,6 +57,30 @@ overrides the default behavior described above in [Using Auth-Auth
 Token](/docs/agent/caching#using-auto-auth-token), and instead ignores any
 existing Vault token in the request and instead uses the auto-auth token.
 
+## Persistent Cache
+
+Vault Agent can restore tokens and leases from a persistent cache file created by a previous 
+Vault Agent process. The persistent cache is a BoltDB file that includes tuples encrypted 
+by a generated encryption key. The encrypted tuples include the Vault token used to retrieve 
+secrets, leases for tokens/secrets, and secret values.
+
+-> **Note:** Vault Agent Caching persistent cache will only restore _leased_ secrets. Secrets 
+that are not renewable, such as KV v2, will not be persisted.
+
+In order to use Vault Agent persistent cache, auto-authentication must be used. During the restoration 
```

Comment:
Nit: [we refer](https://www.vaultproject.io/docs/agent/autoauth) to the feature as auto-auth, so it might be good to be consistent and do the same here.

Reply:
Done!